MAUI/DataForm/migration.md

```diff
@@ -228,6 +228,11 @@ this.Content = dataForm;
 <td>Defines the constants for the ValidationMode that specifies when the user input should be validated for the data object.</td>
 </tr>
 
+<tr>
+<td>{{'[TextInputLayoutSettings](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.TextInputLayoutSettings.html)'| markdownify }}</td>
+<td>TextInputLayoutSettings</td>
+<td>Represents the class to settings for layout type is TextInputLayout.</td>
```

Comment:
<td>Represents the class to settings for layout type to TextInputLayout.</td>

Reply:
Content changed